lua/gitlab/actions/create_mr.lua

```diff
@@ -224,22 +188,42 @@ M.open_confirmation_popup = function(mr)
   local function exit()
     local title = vim.fn.trim(u.get_buffer_text(M.title_bufnr))
     local description = u.get_buffer_text(M.description_bufnr)
-    local target = vim.fn.trim(u.get_buffer_text(target_popup.bufnr))
+    local target = vim.fn.trim(u.get_buffer_text(M.target_bufnr))
```

Comment:
Changes like this were not really necessary, since both `target_popup.bufnr` and `M.target_bufnr` are the same number, but I thought it makes the code easier to read if buffer numbers are accessed in the same way.